### ISSUE TITLE
ACME doc missing in provisioners.md

### ISSUE THREAD

**Issue by thomic** [smallstep/certificates#188]
In https://github.com/smallstep/certificates/blob/master/docs/provisioners.md you are missing how to set up the ACME interface / provisioner.

It would be nice to have this as well.

**Comment by mmalone** [comment]
Good call. These docs are moving to the website, and we'll definitely make sure ACME provisioner docs are on the list.

In the meantime, if you (or anyone else who finds this issue) are blocked on this, ACME provisioner setup is documented pretty well in the blog post announcing the feature: https://smallstep.com/blog/private-acme-server/

**Comment by Afterglow** [comment]
The example in the blog post doesn't work it asks you to run `step ca provisioner add acme --type ACME` but trying to run that yields an error

```
step ca provisioner add acme --type ACME
'step ca provisioner add' requires the '--ca-config' flag
```

The docs in docs/acme.md provide basically the same command but also link to https://smallstep.com/docs/cli/ca/provisioner/add/ which suggests running the command with the `--ca-config` flag. But I can't work out what the argument should be it says `ca.json` but I can't find any information on what I should put in that file?

**Comment by dopey** [comment]
Apologies for the radio silence. The `ca.json` file is generated on the server where you are running the `CA`, when you run `step ca init`. It is stored in `$(step path)/config/ca.json`. 

The `provisioner add` command must be run from the same server where the `step-ca` is running because it modifies the configuration file directly. After the command completes successfully you need to restart (or SIGHUP) your `step-ca` so that it will re-parse the config file.

**Comment by dopey** [comment]
If you're still having trouble please come hit us up in our [gitter](https://gitter.im/smallstep/community). We're usually a bit better about responding to questions through that forum.
